src/04_pinning/01_chapter.md

```diff
@@ -15,7 +15,7 @@ works. Consider the following code:
 ```rust
 let fut_one = ...;
 let fut_two = ...;
-async move {
+async {
```

Comment:
why was this changed? the previous syntax was as-intended.

Reply:
my bad.  reverting it